bundle/edu.gemini.sp.vcs/src/main/scala/edu/gemini/sp/vcs2/TemplateNumberingCorrection.scala

```diff
@@ -21,8 +21,14 @@ class TemplateNumberingCorrection(lifespanId: LifespanId, nodeMap: Map[SPNodeKey
     def correct(tokens: List[(VersionToken, SPNodeKey)]): TryVcs[MergePlan] = {
       def updateToken(tgLoc: TreeLoc[MergeNode], vt: VersionToken): TryVcs[TreeLoc[MergeNode]] =
         tgLoc.getLabel match {
-          case m@Modified(_, _, tg: TemplateGroup, _, _) =>
-            TryVcs(tgLoc.modifyLabel(_ => m.copy(dob = tg.copy <| (_.setVersionToken(vt)))))
+          case m@Modified(_, nv, tg: TemplateGroup, _, _) =>
+            if (tg.getVersionToken == vt)
```

Comment:
This might read better as two cases with a guard on the first one. But it's fine as-is.